draft-ietf-quic-http.md

```diff
@@ -1377,14 +1378,18 @@ the DUPLICATE_PUSH.
 
```

Comment:
Now that you're discussing H2 reserved types in this section, maybe it should be {#frame-reserved}?

Reply:
Perhaps, but those anchors don't appear in the final doc, so I care less.  Still, there are only three references, so it's a minor change.